.travis.yml

```diff
@@ -115,8 +108,7 @@ install:
   # for python 3 we dont install everything yet..
   - |
       [ $TESTS != py3 ] || \
-      pip install -e slave \
```

Comment:
As far as I can tell, there aren't currently any compatibility tests. We can deal with this if/when we add some.